examples/ink/interact.eg.ts

```diff
@@ -51,7 +55,11 @@ await contract
   .sent()
   .dbgStatus("Flip:")
   .inBlockEvents()
-  .pipe(contract.filterContractEvents)
+  .pipe((events) =>
+    Rune.resolve(events).map((events) => events.filter((e) => RuntimeEvent.isContracts(e.event)))
+  )
+  // .pipe(contract.filterContractEvents)
```

Comment:
@harrysolovay  Pattern development question
What would be better?
- filter using the `RuntimeEvent.isContracts`
- filter hardcoding the `Contracts` value (coming from [`construct_runtime!`](https://github.com/paritytech/substrate-contracts-node/blob/fbc28a7ad4b4a58bc9d8bc848f5f3c46018ec81a/runtime/src/lib.rs#L386)) in the `InkRune`